### ISSUE TITLE
My thread didn't appear

### ISSUE THREAD

**Issue by smrose** [scripting/blue.feedland#21]
I posted a 2-post thread from @sm-rose.bsky.app, an account I use just for testing, and in the UI everything seemed to go okay - the expected 1s delay between posts, for example - but the post hasn't appeared, despite I repeated it twice. I used an app password. Possibly something in the logs will suggest what failed.

**Comment by scripting** [comment]
Can you [open the JavaScript console](https://www.google.com/search?q=open+javascript+console), try again, and see if there are any error messages there.

**Comment by smrose** [comment]
Yes, there are 500s for each post in the thread, spaced the expected ~1s apart.

![thread](https://github.com/user-attachments/assets/5e262065-dafd-43a7-9e19-02568798e00d)

**Comment by smrose** [comment]
Is it worth noting that I created both a 'rose' and an 'smrose' account this morning (Pacific time here). I create 'rose' using my first-class account before realizing that it would be better to use my test account as I didn't have an actual thread to post. Then I posted using smrose without ever posting from rose. And that was to be the first post from the smrose account, though I since created a regular post before resubmitting the thread.
